backend/scripts/smoke-stream-mode.sh

```
@@ -0,0 +1,249 @@
+
+# 8. Empty list → falls back to default
+smoke_test "08_empty_list" \
+  "{\"input\":{\"messages\":[{\"role\":\"user\",\"content\":\"$PROMPT\"}]},\"model\":\"$MODEL\",\"stream_mode\":[]}" \
+  200 "messages"
+
+echo ""
+echo "--- Tool-calling tests (read AGENTS.md + append date) ---"
+echo ""
+
+# File content for tool tests — excerpt of AGENTS.md passed via input.files
+AGENTS_MD_CONTENT="# AGENTS.md\n\nThis file provides guidance to AI agents.\n\n## Project Overview\n\nbackend: python, uv, fastapi, langchain\nfrontend: typescript, vite, react, shadcn\n\n## Code Style\n\n- Use Python 3.12+ features\n- Follow PEP 8 conventions\n- Type hints required\n"
```

Shellcheck (0.11.0)

[warning] AGENTS_MD_CONTENT appears unused. Verify use (or export if used externally).

(SC2034)